models/nextsim/dg/namelist.py

```diff
@@ -12,17 +12,17 @@ def make_namelist(path='.', **kwargs):
     # read the config file
     model_config = configparser.ConfigParser()
     model_config.optionxform = str
-    model_config.read('template.cfg')
+    model_config.read(model_config_file)
```

Comment:
Here, the absolute path has to be given for the `model_config_file`. I introduced it from the `nextsim.dg.yml`.